Wikipedia/UI-V5/WMFLocationManager.m

```diff
@@ -6,8 +6,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-static CLLocationDistance WMFMinimumDistanceBeforeUpdatingLocation = 1.0; //meters before we update location
-
 @interface WMFLocationManager ()<CLLocationManagerDelegate>
```

Comment:
Oh, and removing duplicate state (making `lastHeading` and `lastLocation` actual proxy methods that return the underlying location manager's `heading` and `location`).